### PR TITLE
Assignments/number functions

### PR DESCRIPTION
**Pull request by AWalnut1**
cool stuff, was fun.

### CODE REVIEW

File: assignments/number-functions/code.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+function add (a,b) {
```
Comment (gigamonkey) [inline assignments/number-functions/code.js:1]: Lots of kids seem to have picked up this syntax somewhere. Which is fine: it's legal Javascript. However I'm trying to teach you a slightly different syntax and want you to get comfortable with it.

```suggestion
const add = (a, b) => {
```

Can you convert these functions to that syntax to make sure you're comfortable with it?

**Review by gigamonkey — CHANGES_REQUESTED** [review]
Overall looks good! However I would like you to switch to the other form of function definition. Then I'll re-review and most likely approve this to be merged.

File: assignments/number-functions/code.js (hunk 1 above)
Comment (AWalnut1) [reply assignments/number-functions/code.js]: done, sorry i havent seen this untill today

**Review by gigamonkey — APPROVED** [review]
Whoops. Missed that you had updated. This is ready to merge.

**Pull request merged**